pkg/security/config/config.go

```diff
@@ -181,11 +181,6 @@ func NewConfig() (*Config, error) {
 }
 
 func NewRuntimeSecurityConfig() (*RuntimeSecurityConfig, error) {
-	dumpDuration := time.Duration(coreconfig.SystemProbe.GetInt("runtime_security_config.activity_dump.cgroup_dump_timeout")) * time.Minute
```

Comment:
Why was this removed ?

Reply:
Both were moved to https://github.com/DataDog/datadog-agent/pull/17188/files#diff-9e22888ba45a2afd83b1a45a4a670f3697a07aa9860351f01bc252994cd0258d